### ISSUE TITLE
Research different AI types

### ISSUE THREAD

**Issue by Tims777** [amosproj/amos2023ws06-sales-lead-qualifier#23]
According to the planning document: `expected value = life-time value of lead x probability of the lead becoming a customer`

Try keeping that in mind, while reading up on different AI types, but also stay critical and ask yourself if it is really feasible to implement the given formula with AI or whether another approach might yield better results.

**Comment by ultiwinter** [comment]
Since our AI model wants to predict an expected (continuous) value, this is typically a regression problem. In regression analysis, you aim to predict a continuous numerical value based on input features we are yet to collect. In the context of predicting a customer's expected value.
As a rule of thumb, it's often a good practice to start with simple ML models before attempting more complex neural network models. This approach allows us to establish a baseline for our problem and gain insights into the data without immediately diving into the complexity of deep learning. And of course we want every model and achieve the highest performance we get.

For ML models and NN models, there are multiple techniques we can leverage in order to increase the performance:

1. Different initialization techniques (in case of NN: Xavier and He initialization techniques)
2. L1/L2 regularization (weight decay): reduced overfitting and regularizes the model
3. Batch Normalization
4. Dropout (in case of NN)
5. Architecture choice of NN (Depth and width)
6. Optimizers

Feel free to add more
